### PR TITLE
I/add carthage

### PR DESCRIPTION
**Pull request by kbujak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +26,24 @@ Then, you need to run below comand to install framework into your project:
 ```ruby
 pod install
 ```
+#### From Carthage
```
Comment (Szaq) [inline README.md]: I'd omit `From` and keep just `Carthage`, `Cocoapods`, etc...

File: README.md [hunk 2]
```diff
@@ -25,6 +26,24 @@ Then, you need to run below comand to install framework into your project:
...
+[Carthage](https://github.com/Carthage/Carthage) is decentralized dependency manager which allows to build dependencies.
```
Comment (Szaq) [inline README.md]:
```suggestion
[Carthage](https://github.com/Carthage/Carthage) is decentralized dependency manager which allows you to keep your dependencies in compiled format. 
```

File: README.md [hunk 3]
```diff
@@ -25,6 +26,24 @@ Then, you need to run below comand to install framework into your project:
...
 pod install
 ```
+#### From Carthage
+[Carthage](https://github.com/Carthage/Carthage) is decentralized dependency manager which allows to build dependencies. 
+1. To install it, add following line to your Cartfile:
+```ruby
+github 'SwingDev/RxSwiftAutoRetry'
+```
+2. Next, run `carthage update`
+3. On your application targets’ **Build Phases** tab, in the **Link Binary With Libraries** section, drag and drop `RxAtomic.framework`, `RxSwift.framework` and `RxSwiftAutoRetry.framework` from the Carthage/Build folder on disk.
```
Comment (Szaq) [inline README.md]: Is `RxAtomic` needed ?

File: README.md [hunk 4]
```diff
@@ -25,6 +26,24 @@ Then, you need to run below comand to install framework into your project:
...
+4. On your application targets’ Build Phases settings tab, click the + icon and choose New Run Script Phase. Create a Run Script in which you specify your shell (ex: /bin/sh), add the following contents to the script area below the shell:
```
Comment (Szaq) [inline README.md]:
```suggestion
4. On your application targets’ **Build Phases** settings tab, click the + icon and choose **New Run Script Phase**. Create a Run Script in which you specify your shell (ex: /bin/sh), add the following contents to the script area below the shell:
```

File: README.md (hunk 3 above)
Comment (kbujak) [reply README.md]: RxSwift needs RxAtomic. I found on some forum that if framework A needs framework B which has dependency C, then A need both B and C.

**Pull request merged**